hubblestack/extmods/nova_v2_modules/fdg.py

```diff
@@ -0,0 +1,77 @@
+# -*- encoding: utf-8 -*-
+
+import logging
+import os
+import re
+
+from hubblestack.utils.hubble_error import AuditCheckValdiationError
+from hubblestack.utils.hubble_error import AuditCheckFailedError
+from salt.exceptions import CommandExecutionError
+
+log = logging.getLogger(__name__)
+
+def execute(check_id, audit_check):
+    """Execute single check
+
+    Arguments:
+        check_id {str} -- Unique check id
+        audit_check {str} -- Dictionary of an individual check implementation
+
+    Returns:
+        dict -- dictionary of result status and output
+
+    Raises:
+        AuditCheckFailedError -- In case of error
+    """
+
+    fdg_file = audit_check['file']
+    starting_chained = audit_check.get('starting_chained')
```

Comment:
What is this parameter used for ? Can we have a test case explaining this feature as well.